examples/enable/basic_move.py

```diff
@@ -3,10 +3,11 @@
 """
 from __future__ import print_function
 
-from enable.example_support import DemoFrame, demo_main
-
 from traits.api import Float
-from enable.api import Component, Pointer, Window
+from traits.api import HasTraits, Instance
+from traitsui.api import Item, View
+
+from enable.api import Component, Pointer, ComponentEditor
 
 
 class Box(Component):
```

Comment:
same comments about the `resizable` trait as before and about the box colors as before (basic_draw).